### PR TITLE
Life situations

### PR DESCRIPTION
**Pull request by loumoore**
consolidates several questions about life situations

### CODE REVIEW

**Comment by loumoore** [comment]
this change is simpler than the number of changed files would suggest, it's mostly deleting things. but if this is unwieldy for an async review I'd be happy to hop on a zoom and walk someone through the changes.

File: spec/features/web_intake/new_joint_filers_spec.rb [hunk 1]
```diff
@@ -84,14 +84,8 @@
...
-    expect(page).to have_selector("h1", text: "Were you a full-time student in 2019?")
-    click_on "No"
-    expect(page).to have_selector("h1", text: "In 2019, were you in the United States on a Visa?")
-    click_on "No"
-    expect(page).to have_selector("h1", text: "In 2019, did you have a permanent disability?")
-    click_on "Yes"
-    expect(page).to have_selector("h1", text: "In 2019, were you legally blind?")
-    click_on "No"
+    expect(page).to have_selector("h1", text: "Select any situations that were true for you in 2019")
+    click_on "Continue"
```
Comment (bengolder) [inline spec/features/web_intake/new_joint_filers_spec.rb:88]: One of these feature specs should check all of the boxes, while the other should do the least.

File: spec/features/web_intake/new_joint_filers_spec.rb [hunk 2]
```diff
@@ -136,14 +130,8 @@
...
-    expect(page).to have_selector("h1", text: "Was your spouse a full-time student in 2019?")
-    click_on "No"
-    expect(page).to have_selector("h1", text: "In 2019, was your spouse in the United States on a Visa?")
-    click_on "No"
-    expect(page).to have_selector("h1", text: "In 2019, did your spouse have a permanent disability?")
-    click_on "Yes"
-    expect(page).to have_selector("h1", text: "In 2019, was your spouse legally blind?")
-    click_on "No"
+    expect(page).to have_selector("h1", text: "Select any situations that were true for your spouse in 2019")
+    click_on "Continue"
```
Comment (bengolder) [inline spec/features/web_intake/new_joint_filers_spec.rb:134]: One of these feature specs should check all of the boxes, while the other should do the least.

File: spec/features/web_intake/new_single_filer_spec.rb [hunk 3]
```diff
@@ -90,19 +90,12 @@
...
-    expect{ track_progress }.to change { @current_progress }.by_at_least(1)
-    click_on "No"
+    click_on "Continue"
 
-    expect(page).to have_selector("h1", text: "In 2019, did you have a permanent disability?")
-    click_on "Yes"
-    expect(page).to have_selector("h1", text: "In 2019, were you legally blind?")
-    click_on "No"
     expect(page).to have_selector("h1", text: "Have you ever been issued an IP PIN because of identity theft?")
+    expect{ track_progress }.to change { @current_progress }.by_at_least(1)
```
Comment (bengolder) [inline spec/features/web_intake/new_single_filer_spec.rb:98]: Thanks for keeping this & moving it

**Pull request merged**